src/prefect/events/schemas/automations.py

```diff
@@ -429,4 +432,112 @@ class AutomationCore(PrefectBaseModel, extra="ignore"):
 
 
 class Automation(AutomationCore):
-    id: UUID = Field(..., description="The ID of this automation")
+    id: Optional[UUID] = Field(default=None, description="The ID of this automation")
+
+    @classmethod
+    @sync_compatible
+    async def create(cls: Type[Self], automation: Type[Self]) -> UUID:
```

Comment:
since `automation` is an instance of `Automation` and not the literal type object `Automation`
```suggestion
    async def create(cls: Type[Self], automation: Self) -> UUID:
```

Reply:
same below

Reply:
on this point though, I find it a little odd to have a classmethod whose `create` method accepts an instance of that same class 🤔

Reply:
Yea, this is too wonky -- maybe you were thinking `Automation(...).create()` with no call args?